storage/innobase/include/srv0srv.h

```diff
@@ -760,6 +760,9 @@ struct export_var_t{
 	ulint innodb_encryption_rotation_pages_flushed;
 	ulint innodb_encryption_rotation_estimated_iops;
 	int64_t innodb_encryption_key_requests;
+
+	/* Number of bulk insert operations */
+	Atomic_relaxed<ulint> innodb_bulk_insert_operation;
```

Comment:
I would suggest the following with no source code comment:
```c++
	Atomic_counter<ulint> innodb_bulk_operations;
```
While we are at this, I think that we should also change `innodb_instant_alter_column` to be of the same type. The two fields could be moved next to each other, because they are logically related.

I think that I previously requested that the counter `innodb_bulk_operations` be also incremented during table-rebuilding native `ALTER TABLE`. The following would seem to be the logical spot for that:
```diff
diff --git a/storage/innobase/handler/handler0alter.cc b/storage/innobase/handler/handler0alter.cc
index 3c5db74e17a..7ed787af22f 100644
--- a/storage/innobase/handler/handler0alter.cc
+++ b/storage/innobase/handler/handler0alter.cc
@@ -7357,6 +7357,7 @@ prepare_inplace_alter_table_dict(
 		ut_d(dict_table_check_for_dup_indexes(user_table,
 						      CHECK_PARTIAL_OK));
 		if (ctx->need_rebuild()) {
+			export_vars.innodb_bulk_insert_operations++;
 			ctx->new_table->acquire();
 		}
 
```